### PR TITLE
DOP-2392: Add UnifiedFooter to docs-nav

### PR DESCRIPTION
**Pull request by rayangler**
### Stories/Links:

DOP-2392

### Staging Links:

[Realm staging](https://docs-mongodbcom-staging.corp.mongodb.com/master/realm/raymundrodriguez/DOP-2392/) - with consistent nav feature flag on
[Realm staging](https://docs-mongodbcom-staging.corp.mongodb.com/master/realm/raymundrodriguez/DOP-2392-con-nav-off/) - without feature flag

### Notes:
* Adds `UnifiedFooter` to `DocumentBody`.
* Had to update the positioning of our `RightColumn` that hosts the "On This Page"/`Contents` component to avoid the component from overlapping with the new footer.
* Also updated some props for the `Contents` and `Widgets` files to avoid unnecessary prop errors.

### CODE REVIEW

File: src/components/Contents.js [hunk 1]
```diff
@@ -61,7 +61,7 @@ const StyledContents = styled('div')`
 `;
 const ContentsListItem = ({ children, depth, id, isActive }) => (
   <ListItem isActive={isActive}>
-    <StyledLink to={`#${id}`} isActive={isActive} depth={depth}>
+    <StyledLink to={`#${id}`} depth={depth}>
```
Comment (mango-db) [inline src/components/Contents.js:64]: This is an unrelated clean-up, right? 🗑️

File: src/components/Widgets/index.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ import PropTypes from 'prop-types';
 import { FeedbackProvider, FeedbackForm, FeedbackTab, useFeedbackData } from './FeedbackWidget';
 import { isBrowser } from '../../utils/is-browser';
 
-const Widgets = ({ children, location, pageOptions, pageTitle, publishedBranches, slug }) => {
+const Widgets = ({ children, pageOptions, pageTitle, publishedBranches, slug }) => {
```
Comment (mango-db) [inline src/components/Widgets/index.js:6]: `location` disappeared but is still a required prop?

**Review by mango-db — COMMENTED** [review]
something something we've finally located BIG FOOTER

image

### LGTM:
- [x] thank you for saving us from `publishedBranches` prop warnings 🙇 [This line](https://github.com/mongodb/snooty/blob/master/src/components/Widgets/FeedbackWidget/useFeedbackData.js#L10) seems to indicate that this is truly an unrequired prop
- [x] Sensible toggling between `Footer` and `UnifiedFooter`
- [x] Successfully built w/ `UnitedFooter` using feature flag
- [x] `UnifiedFooter` [Staging](https://docs-mongodbcom-staging.corp.mongodb.com/master/realm/cgoecknerwald/DOP-2392/sdk/dotnet/) looks fine on desktop, mobile, among Chrome/Firefox/Safari
- [x] Regular `Footer` still looks fine (esp. with `RightColumn` play) 
- [ ] TFW your footer has dark mode before you do 😞  

### Questions:
- Is toggling between English / Portuguese in the UnifiedFooter actually supposed to do anything right now?

**Review by casthewiz — COMMENTED** [review]
This is looking good - we can pass this `hideLocale={true}` as a prop to the footer to disable the localization options.

File: src/components/Contents.js (hunk 1 above)
Comment (rayangler) [reply src/components/Contents.js:64]: Yup, we weren't using the prop for anything and I wanted to get rid of the console warning

**Comment by rayangler** [comment]
Updated the staging link to reflect recent changes.

I updated the consistent-nav package to the latest version here to gain access to the `hideLocale` prop and to have the top nav now be compatible with our grid layout

**Review by allisonmui9 — APPROVED** [review]
lgtm! thanks raymund!!

**Review by mango-db — APPROVED** [review]
LGTM
